eng/common/templates-official/job/publish-build-assets.yml

```diff
@@ -68,7 +68,7 @@ jobs:
     - checkout: self
       fetchDepth: 3
       clean: true
-      
```

Comment:
Revert changes in this file. The extra whitespace are from dotnet/runtime main

Reply:
Thanks, I was reviewing with whitespace changes hidden.